conformance-tests/src/assertions/java/org/jspecify/conformance/tests/irrelevantannotations/notnullmarked/Other.java

```diff
@@ -65,6 +65,12 @@ void catchParameters() {
       // test:irrelevant-annotation:NonNull
```

Comment:
Should there be different tests for "irrelevant" vs. "intrinsically non-null"?

Reply:
We've been using `irrelevant-annotation` for all cases where the annotation is unrecognized or inapplicable, without requiring tools to distinguish.